hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/AutoAccountCreationSuite.java

```diff
@@ -546,7 +549,8 @@ final HapiSpec multipleTokenTransfersSucceed() {
         final var initialTokenSupply = 1000;
         final var multiTokenXfer = "multiTokenXfer";
 
-        return defaultHapiSpec("multipleTokenTransfersSucceed")
+        return propertyPreservingHapiSpec("multipleTokenTransfersSucceed")
+                .preserving("entities.unlimitedAutoAssociations", FALSE)
```

Comment:
I'm not sure if this is correct, I believe that in this case preserving only accepts list of names of configs. But seems like this change assumes that preserving accepts CONFIG_NAME -> CONFIG_VALUE but this is not the correct. Might be mistaken for overriding(CONFIG_NAME, CONFIG_VALUE).